File: IClientNetworkingUtilsSerialized.cs

using System;
using System.Runtime.InteropServices;

namespace SteamStructs {
    [StructLayout(LayoutKind.Sequential, Pack = 1)]
    internal struct IClientNetworkingUtilsSerialized
    {
        public IntPtr GetNetworkConfigJSON_DEPRECATED;
        public IntPtr GetLauncherType;
        public IntPtr TEST_ClearCachedNetworkConfig;
    }
}

File: IClientNetworkingUtilsSerialized.cc
class IClientNetworkingUtilsSerialized {
	virtual unknown_ret GetNetworkConfigJSON_DEPRECATED(void*, unsigned int, char const*) = 0;
	virtual unknown_ret GetLauncherType() = 0;
	virtual unknown_ret TEST_ClearCachedNetworkConfig() = 0;
};
